### ISSUE TITLE
Masked face recognition

### ISSUE THREAD

**Issue by AlonZolfi** [JDAI-CV/FaceX-Zoo#65]
Hey everyone,

I want to use the pre-trained masked face recognition model (trained on the MegaFace-Mask) but I can't find any example in the entire repo on how to do it (image preprocessing, weights file, etc.) in the simple pytorch manner, like:

`
model = torch.load(...)
img = preprocess_image(original_image)
output = model(img)
`

Thanks!

**Comment by wang21jun** [comment]
Just add two configurations in train.sh
--resume \
--pretrain_model 'Path of the pretrained model' \
